src/pages/home/report/ReportActionsView.js

```diff
@@ -196,7 +196,13 @@ class ReportActionsView extends React.Component {
     updateSortedReportActions(reportActions) {
         this.sortedReportActions = _.chain(reportActions)
             .sortBy('sequenceNumber')
-            .filter(action => action.actionName === 'ADDCOMMENT' || action.actionName === 'IOU')
+            .filter((action) => {
+                const message = _.last(lodashGet(action, 'message', null));
```

Comment:
hmmm not 100% sure but I think that maybe this should be:

```suggestion
                const message = _.first(lodashGet(action, 'message', null));
```

Reply:
Because the newest message will be `message[0]`, right?

Reply:
I don't know for sure, I copied the logic from [here](https://github.com/Expensify/Expensify.cash/blob/9075af921b33e1760af1464bc601e8f54ec7e25e/src/pages/home/report/ReportActionContextMenu.js#L67-L69). I think there is only one message, so it shouldn't matter, last and first might be the same?

Reply:
Hmmm okay I think both the here and the location you linked should be changed to `_.first`, because when editing comments the newest version is being stored at `message[0]`